processor.go

```diff
@@ -245,8 +246,22 @@ func (p *queryProcessor) measure() {
 			// Now add all of the data to the measurer.
 			for j := range bucket.sourceRows {
 				row := bucket.sourceRows[j]
+
+				// If measurer is cardinality, we can +1 not worrying about field value.
+				if (reflect.TypeOf(m) == reflect.TypeOf(&cardinality{})) {
```

Comment:
You should do this for a type check instead:`if _, ok := m.(*cardinality); ok {`

But.... this doesn't really apply, since cardinality should be unique, I'm not sure this shortcut works any more.